pages/index.html

```diff
@@ -4,9 +4,49 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="styles/style.css" type="text/css">
+  <link rel="icon" type="image" href="images/favicon.ico">
+  <!-- The navigation menu -->
+  <div class="navbar">
+    <a class="active" href="index.html">Home</a> 
+    <a href="about.html">About</a> 
+    <a href="portfolio.html">Portfolio</a> 
+  </div>
+  <title>Kristel's Website!</title>
 </head>
+
 <body>
-  
+  <header class="header">
+    <div class="overlay">
+      <h1 class="header-title">Welcome to<br>Kristel's page!</h1>
+    </div>
+  </header>
+
+  <section class="content">
+    <div class="card">
+      <img class="card-image" src="https://i.imgur.com/G2U8Trs.jpg">
```

Comment:
Your `img` tags should  have an `alt` attribute that way screen readers and bots will have an idea about the image.